power-interruption-logging-system/main.go

```diff
@@ -33,8 +33,7 @@ func (i interruption) is_scheduled(list []interruption) bool {
 
 // function to check if interruption is scheduled using the date
 func is_scheduled_date(list []string, date string) bool {
```

Comment:
all file names should be in lowercase snakecase, Go source files and directories should use underscores “” instead of dashes “-”.